airflow/config_templates/default_airflow.cfg

```diff
@@ -489,9 +489,11 @@ basedn = dc=example,dc=com
 cacert = /etc/ca/ldap_ca.crt
 search_scope = LEVEL
 basedn_group= dc=example,dc=com
-group_filter=
-group_name_attr=
+
 superuser_group=
```

Comment:
adding a comment here would be nice. Sth like:

> declare new empty configuration variables; these variables can be overridden during application load time (with environment variables, e.g.: when using docker-compose, set these variables in `docker-compose.yml` file)
see https://airflow.apache.org/howto/set-config.html

Reply:
I understand the idea behind the comment. But since it is a common practice to either use an environment file or docker-compose file for airflow deployment this might seem a bit repetitive ? I think we should explain more the idea behind the usage of those variable ?

Reply:
@galak75 Any feedback?

Reply:
A comment has been added. LGTM